apps/web/src/app/components/Button.tsx

```diff
@@ -0,0 +1,60 @@
+import { classed, deriveClassed, ComponentProps } from "~/utils/classed";
+import { useHomeStore } from "~/lib/home/home-store";
+import { createOrGetFx } from "~/styles/sound";
+
+type PropsV2 = {
+  onClick?: () => void;
+  children?: React.ReactNode;
+  variant?: "primary" | "secondary" | "link"; // Represents the color of the button
+  size?: "small" | "medium";
+};
+
+const StyledButton = classed.button(
+  "rounded-md border-2  text-white mb-7 inline-block relative text-center",
+
+  {
+    variants: {
+      variant: {
+        primary: "bg-accent-yellow-regular",
+        secondary: "bg-accent-gray-regular",
+        link: "bg-transparent",
+      },
+      size: {
+        small: "w-[150px] h-[27px]",
```

Comment:
we shouldn't be hardcoding widths and heights for buttons otherwise they're not very usable

please use `sm`/`md`/`lg` as size variants